application/common/helpers/Utils.php

```diff
@@ -0,0 +1,120 @@
+<?php
+namespace common\helpers;
+
+use yii\base\Security;
+
+class Utils
+{
+    const FRIENDLY_DT_FORMAT = 'l F j, Y g:iA T';
+
+    /**
+     * @param int $length
+     * @return string
+     */
+    public static function generateRandomString($length = 32)
+    {
+        $security = new Security();
+        return $security->generateRandomString($length);
+    }
+
+    /**
+     * Return a random string of numbers
+     * @param int $length [default=4]
+     * @return string
+     * @throws \Exception
+     */
+    public static function getRandomDigits($length = 4)
+    {
+        $result = '';
+        while (strlen($result) < $length) {
```

Comment:
Now that we're using PHP >= 7, this whole thing could probably be simplified to instead use [random_int()](http://php.net/random_int). Something like this might work:
```php
$result = '';
for ($i = 0; $i < $length; $i++) {
    $result .= random_int(0, 9);
}
return $result;
```

Reply:
Thanks!